routes/users.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router();
+const { pool } = require('../server/db');
+
+router.use(express.json());
+
+router.post('/', async (req, res) => {
+  try {
+    const data = req.body;
+    const newUser = await pool.query(
+      `INSERT INTO users VALUES('${data.id}', '${data.role}', '${data.firstName}' , '${data.lastName}', '${data.phoneNumber}', '${data.email}') RETURNING *;`,
+    );
+    res.status(200).json(newUser.rows[0]);
+  } catch (err) {
+    console.log(err);
+    res.status(400).send(err.message);
+  }
+});
+
+router.get('/:userId', async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const getUser = await pool.query(`SELECT * FROM users WHERE id = '${userId}'`);
+    res.status(200).json(getUser.rows[0]);
+  } catch (err) {
+    console.log(err);
+    res.status(400).send(err.message);
+  }
+});
+
+router.put('/:userId', async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const data = req.body;
+    const updateUser = await pool.query(`UPDATE users SET 
```

Comment:
see above re: variable naming

Reply:
Nit-pick: can we name these variables something like `updatedUser` (as an adjective instead of a verb)?